src/sql/sqlops.proposed.d.ts

```diff
@@ -303,7 +309,7 @@ declare module 'sqlops' {
 		initializeModel<T extends Component>(root: T): Thenable<void>;
 	}
 
-	export namespace dashboard {
+	export namespace modelView {
```

Comment:
I don't know if `modelView` is the most intuitive name from this namespace.  Maybe something like `uiBuilder` or something else that is more clear of what is the purpose of this set of classes.